core/templates/pages/exploration-player-page/services/current-interaction.service.spec.ts

```diff
@@ -115,25 +122,45 @@ describe('Current Interaction Service', () => {
     currentInteractionService.registerPresubmitHook(hookB);
 
     currentInteractionService.setOnSubmitFn(() => {});
-    currentInteractionService.onSubmit(null, null);
+    currentInteractionService.onSubmit(DUMMY_ANSWER, interactionRulesService);
 
     expect(hookStateA).toEqual(1);
     expect(hookStateB).toEqual(3);
 
     currentInteractionService.clearPresubmitHooks();
-    currentInteractionService.onSubmit(null, null);
+    currentInteractionService.onSubmit(DUMMY_ANSWER, interactionRulesService);
 
     expect(hookStateA).toEqual(1);
     expect(hookStateB).toEqual(3);
   });
 
   it('should throw error on submitting when submitAnswerFn is null', () => {
+    let interaction = {
+      customization_args: {},
+      answer_groups: [],
+      default_outcome: {},
+      confirmed_unclassified_answers: [],
+      id: null
+    } as unknown as Interaction;
```

Comment:
You're adding `unknown` here, don't we want to get rid of it?

Reply:
ditto below

Reply:
These properties are not really of use here, they are just present for being passed as an argument(instead of `null` which causes type errors) and something else is being tested. Should these be filled with unwanted values?

Reply:
Removed `unknown`